### ISSUE TITLE
A way to restart the mine

### ISSUE THREAD

**Issue by Rynald4d** [4dmud/mud#15]
The mine is static atm, what if we add an option to config that determines if the mine resets on the next copyover? People could then tunnel their hearts out and discover Moria all over. It wouldn't reset until the flag is set again by an imm some time in the future.

**Comment by Tocharaeh** [comment]
good idea to freshen things up.

**Comment by J00bei** [comment]
I thought the mines always reset on copyovers? I know Moria can only be accessed by the lower 3 levels but it's still very random, What I would change is from what I remember if you enter Moria from through the mines 3 times in a boot on the 3rd time the entrance to Moria will collapse, and no longer be accessable. 

I've also had times after a fresh reset, where my RNG was so bad that it was impossible to actually get into Moria. I think once a player finds Moria in a reset they should be able to continue to reaching it.

**Comment by lfaoro** [comment]
I agree, the entrance to the mine should never collapse and stay that way until a copyover: this makes the player experience with the zone very bad.
